src/CommandingOfficers/Tech.java

```diff
@@ -194,10 +194,15 @@ protected void revert(MapMaster gameMap)
       if( unitsOverCharged.containsKey(myCommander) )
       {
         // End Overcharge. Any units who still have > MAXIMUM_HP get reset to max.
+        final int maxPercent = UnitModel.MAXIMUM_HP*10;
         for( Unit u : unitsOverCharged.get(myCommander) )
         {
-          if( u.getPreciseHP() > UnitModel.MAXIMUM_HP )
-            u.alterHP(10);
+          final int preciseHP = (int) (u.getPreciseHP() * 10);
+          if( preciseHP > maxPercent )
```

Comment:
Comparing an HP value with a percent value looks wrong. Looks like just a variable-naming thing, not an actual mis-comparison, but it smells as is.

Reply:
"Precise HP" and "percent health" are the same thing.
On that note, getting a double, multiplying by 10, and casting to int kinda gives me heartburn. Can/should I just introduce a `getHealthPercent()` that returns the raw value?

Another option that's 100% outside the scope of this PR is to move to using % health everywhere but display, and leave the accessors in place to round it where needed (almost everywhere).

Reply:
I mean, we could. And it would frankly make sense to just move to the 0-100 health value everywhere except in the display. I've had the heebie-jeebies about how we've been handling health and HP as two different quantities for a while.